fog/view/server/tests/streaming_smoke_tests.rs

```diff
@@ -124,9 +124,9 @@ fn test_streaming_integration(omap_capacity: u64, store_count: usize, blocks_per
         .sort_by_key(|rng_record| rng_record.ingest_invocation_id);
     assert_eq!(result.rng_records[0].pubkey, egress_public_key_1);
     assert_eq!(result.rng_records[1].pubkey, egress_public_key_2);
-    assert_eq!(result.tx_out_search_results.len(), 1);
+    assert_eq!(result.fixed_tx_out_search_results.len(), 1);
```

Comment:
:wrench: it seems like these tests steps should still be checking both members (`tx_out_search_results` and `fixed_tx_out_search_results`) for lengths and contents throughout the test.

Reply:
Done.